content/teams/teams.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Teams
+weight: 1
+---
+
+If more than one developer is contributing to a project, it is wiser to set up a team so that everyone could work on the project together. Teams of two are free of charge. It is possible to invite more members by purchasing additional seats. Billing information is available [here](https://codemagic.io/pricing/).
+
+{{<notebox>}}Note that teams who enabled billing before June 6th, 2020, still operate under the previous billing system.{{</notebox>}}
```

Comment:
Teams who enabled billing before June 6th and have not yet migrated to new billing. This migration part, however expressed, is important, otherwise it might confuse already migrated users.